blocks/library/latest-posts/index.php

```diff
@@ -13,13 +13,22 @@
  * @return string Returns the post content with latest posts added.
  */
 function render_block_core_latest_posts( $attributes ) {
-	$recent_posts = wp_get_recent_posts( array(
+	$args = array(
 		'numberposts' => $attributes['postsToShow'],
 		'post_status' => 'publish',
 		'order'       => $attributes['order'],
 		'orderby'     => $attributes['orderBy'],
 		'category'    => $attributes['categories'],
-	) );
+	);
+	$instance = array(
+		'title' => __( 'Recent Posts' ),
```

Comment:
I think this should rather just be an empty string.

Reply:
Or it can just be omitted entirely.

Reply:
That is the instance behaviour, but in the method, we're aiming for parity with there is an empty check that defaults to this.

Reply:
https://github.com/WordPress/wordpress-develop/blob/ba195e435339813afc931db6359a515b92142292/src/wp-includes/widgets/class-wp-widget-recent-posts.php#L48

Reply:
Since the title is not displayed in the block at all, I think it should be omitted entirely or just be an empty string. It's only purpose is to prevent a plugin from filtering the `$args` to attempt to read from `$instance['title']` and encounter an undefined index warning. Really I think that is overkill, so just omitting the title from `$instance` is just fine.

Reply:
Right pushing again :wink: 

Thanks for the patience btw